opencti-platform/opencti-front/src/private/components/assets/devices/DeviceCreation.js

```diff
@@ -149,18 +153,23 @@ class DeviceCreation extends Component {
       //   },
       //   values,
       // );
-    //   variables: {
-    //     input: values,
-    //   },
-    //   setSubmitting,
-    //   onCompleted: (data) => {
-    //     setSubmitting(false);
-    //     resetForm();
-    //     this.handleClose();
-    //     this.props.history.push('/dashboard/assets/devices');
-    //   },
-    //   onError: (err) => console.log('DeviceCreationDarkLightMutationError', err),
-    // });
+      variables: {
+        input: {
+          name: values.name,
+          asset_type: values.asset_type,
+          ports: ports,
+        },
+      },
+      setSubmitting,
+      onCompleted: (data) => {
+        console.log('sfasfasfasfsfRes', data);
```

Comment:
please remove the console.log... these fill the log file and we're forced to remove them